### PR TITLE
feat: navbar in every page as navigation method

### PR DESCRIPTION
**Pull request by assebc**
image

## Issue
https://github.com/b1tsway/b1tsway.github.io/issues/9

## Type of change

Please delete options that are not relevant or select with a "x" the correct option.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist(mark with a "x" the items you made):

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I tested all related functionalities and I didn´t break anything

### CODE REVIEW

**Comment by Mariii-23** [comment]
In other sizes don't work
image
image

**Review by Mariii-23 — CHANGES_REQUESTED** [review]
Fix the layout pls

**Comment by assebc** [comment]
when in mobile version hides the two sections in navbar

File: src/pages/Home/index.tsx [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+import { handleNavigate } from "../../utils/handleNavigate";
 import "./styles.css";
 
 export const Home: FC = () => {
     const navigate = useNavigate();
     
     return (
         <>
-            <button onClick={() => navigate("/contact-us")}>contact us</button>
+            <Header style={{height: "90vh"}}/>
```
Comment (Mariii-23) [inline src/pages/Home/index.tsx:12]: I don't know to what extent in this case to define that the ``min-height`` is "90vh" in the component itself than to be outside.

I don't know if it's correct that we are allowing to pass the style in the component.

**Review by Mariii-23 — CHANGES_REQUESTED** [review]
image
:eyes:

File: src/pages/Home/index.tsx (hunk 1 above)
Comment (assebc) [reply src/pages/Home/index.tsx:12]: this is from a previous PR that was already merged

Comment (assebc) [reply src/pages/Home/index.tsx:12]: either way why not? it is a props as everything else?

**Review by Mariii-23 — APPROVED** [review]

**Pull request merged**